frontend/src/v4/modules/risks/risks.sagas.ts

```diff
@@ -367,18 +367,46 @@ const onCreateEvent = (createdRisk) => {
 	dispatch(RisksActions.saveRiskSuccess(createdRisk[0], false));
 };
 
+const onResourcesCreated = (resources) => {
```

Comment:
The type of attachment is wrong on the receiving end. 
I send an image an it shows like it's a url attachment,

https://github.com/user-attachments/assets/713524f9-d17d-4db5-9867-394ff43f8055